helm/observability-operator/templates/deployment.yaml

```diff
@@ -31,12 +31,15 @@ spec:
         - --management-cluster-pipeline={{ $.Values.managementCluster.pipeline }}
         - --management-cluster-region={{ $.Values.managementCluster.region }}
         # Monitoring configuration
+        - --alertmanager-enabled={{ $.Values.alerting.enabled }}
```

Comment:
I would move this under #alerting configuration but this can be done once all PRs are merged

Reply:
That's already under alerting

Reply:
I meant the comment 😉